src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upbothside;
+    private int downbothside;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int upbothside, int downbothside, int height) {
+        super(color);
+        this.upbothside = upbothside;
+        this.downbothside = downbothside;
+        this.height = height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: IsoscelesTrapezoid");
+        System.out.println("Color: " + getColor());
+        System.out.println("Up both side length: " + upbothside);
+        System.out.println("Down both side length: " + downbothside);
+        System.out.println("Height length: " + height);
+        System.out.println("Area: " + getArea());
+    }
+
+    @Override
+    public double getArea() {
+        return (double) ((upbothside + downbothside) * height) / 2;
```

Comment:
Magic numbers should be avoided or replaced with named constants. The number '2' used in the area calculation formula should be a constant with a descriptive name.